### PR TITLE
[tests-only] [full-ci] Cleanup acceptance tests

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
- Fixup minor typos and grammar (for example "non-current")
- declare types of private properties,function parameters
- remove parameters from calls where the parameters are already the defaults
- remove unneeded curly brackets from string substitutions
- remove unused variables
- remove all remaining references to GraphContext and Graph API (they were for ocis only)
- refactor some code so that variables are guaranteed to be set
- refactor code to use the `??` operator, as suggested by my IDE

Now the code uses the features of PHP 7.4 as much as reasonable.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/AppManagementContext.php [hunk 1]
```diff
@@ -460,7 +453,7 @@ public function assertInstalledVersionOfAppIs(string $appId, string $version):vo
 		Assert::assertEquals(
 			$version,
 			\trim($cmdOutput),
-			"Expected: the installed version of ${appId} should be ${version} but got " . \trim($cmdOutput)
+			"Expected: the installed version of $appId should be $version but got " . \trim($cmdOutput)
```
Comment (SwikritiT) [inline tests/acceptance/features/bootstrap/AppManagementContext.php:456]: should it be `to be` or  `should be` is also correct?
```suggestion
			"Expected: the installed version of $appId to be $version but got " . \trim($cmdOutput)
```

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -355,10 +208,7 @@ public function isRunningForDbConversion():bool {
 		return $this->dbConversion;
 	}
 
-	/**
-	 * @var string
-	 */
-	private $oCSelector;
+	private string $oCSelector;
```
Comment (SwikritiT) [inline tests/acceptance/features/bootstrap/FeatureContext.php:211]: I though we removed all the code related to parallel deployment.

File: tests/acceptance/features/bootstrap/Sharing.php [hunk 3]
```diff
@@ -876,58 +852,6 @@ public function getMimeTypeOfLastSharedFile():string {
...
 
-	/**
-	 * @param string $url
-	 * @param string|null $user
-	 * @param string|null $password
-	 * @param string|null $mimeType
-	 *
-	 * @return void
-	 */
-	private function checkDownload(
```
Comment (SwikritiT) [inline tests/acceptance/features/bootstrap/Sharing.php:887]: Is this function not needed anymore?

File: tests/acceptance/features/bootstrap/FeatureContext.php (hunk 2 above)
Comment (SwikritiT) [reply tests/acceptance/features/bootstrap/FeatureContext.php:211]: Maybe. Qe have an issue for cleaning up ocis related code in core, I'll check if we also have a checklist to clean the code related to parallel deployment as well.

**Review by SwikritiT — APPROVED** [review]

**Comment by phil-davis** [comment]
Similar code cleanup in ocis PR https://github.com/owncloud/ocis/pull/6195

**Pull request merged**